ngx-tools/testing/src/utilities/event-objects.spec.ts

```diff
@@ -50,7 +50,7 @@ describe(`event-objects`, function() {
       const actual: KeyboardEvent = createKeyboardEvent('keydown', KEYS.ENTER, target);
       expect(actual.code).toEqual(KEYS.ENTER.code);
       expect(actual.key).toEqual(KEYS.ENTER.code);
-      expect(actual.keyCode).toEqual(KEYS.ENTER.keyCode);
+      expect(actual.code).toEqual(KEYS.ENTER.code);
```

Comment:
duplicate of line 51
have we removed all references to keyCode?

Reply:
Removed. 

I didn't do a full search as that's not the basis of this PR. I'm honestly not entirely sure how these were missed in the linting PR. They should have been flagged by the deprecation rule when it was implemented.